src/v2/view-builder/views/EnrollProfileView.js

```diff
@@ -150,4 +157,20 @@ export default BaseView.extend({
         false);
     }
   },
+
+  _addIdpView(idpButtons) {
+    // We check the 'idpDisplay' option config to determine whether to render the idp buttons 
+    // above or below the login fields
+    const idpDisplay = this.options.settings.get('idpDisplay');
```

Comment:
Can you add a section to the README about this settings?

Reply:
There is some text in the "classic" README: https://github.com/okta/okta-signin-widget/blob/master/docs/classic.md#openid-connect
This can probably be copied over since the option works in both classic and OIE

Reply:
Done!